doc/telescope.txt

```diff
@@ -872,6 +872,8 @@ builtin.tags({opts})                                          *builtin.tags()*
                                     true)
         {only_sort_tags} (boolean)  if true we will only sort tags (default:
                                     false)
+        {fname_width}    (number)   Defines the width of the filenames section
+                                    (default: 30)
```

Comment:
documentation is autogenerated needs to be added here https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/builtin/init.lua#L102-L117

und both `builtin.tags` and `builtin.current_buffer_tags`